terraform/variables.tf

```diff
@@ -45,3 +45,29 @@ variable "memorystore" {
   type        = bool
   description = "If true, Online Boutique's in-cluster Redis cache will be replaced with a Google Cloud Memorystore Redis cache"
 }
+
+variable "enable_autopilot" {
+  type        = bool
+  description = "If true, GKE cluster is provisioned in autopilot mode. Provisioning in autopilot mode ignores values set in 'gke_node_pool' variable"
+  default     = true
```

Comment:
Thanks for defaulting to autopilot (to avoid changing Online Boutique's Terraform's default behaviour)!